### ISSUE TITLE
Add chain info when you click deposit/withdraw

### ISSUE THREAD

**Issue by maxjuno** [Wasmswap/wasmswap-interface#160]
Example:
If you bought UST on junoswap and never added terra to your keplr you can't withdraw it because you dont have the chain in your keplr.
Would be nice if we can automatically add the chain info when you click deposit/withdraw (approve a keplr popup)
![des](https://user-images.githubusercontent.com/98474118/152827820-d5e8a453-e085-4e17-8e43-0be614398d1f.png)

**Comment by sashimi36** [comment]
@maxjuno yeah that's a weird one. This is probably happening because this chain is experimental and it hasn't been added on Keplr initially. We could provide a way to suggest a chain within the app by supplying a config file for people to fill out in this case and plugging that data in whenever the `enable` chain method throws the app off.

@ben2x4 what are your thoughts?
